static/src/javascripts/projects/commercial/modules/third-party-tags/lotame.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import config from 'lib/config';
+import { isInAuOrNz, isInUsOrCa } from "common/modules/commercial/geo-utils";
+
+export const lotame: ThirdPartyTag = {
+    shouldRun: config.get('switches.lotame', false) && !isInUsOrCa() && !isInAuOrNz(),
```

Comment:
I think we're getting in triple-negation territory:

- IF switch AND NOT (us OR ca) AND NOT (au OR nz)
- IF switch AND NOT(us OR ca OR au OR nz)

Reply:
simplified to 
`config.get('switches.lotame', false) && !(isInUsOrCa() || isInAuOrNz())`